browser/resources/settings/brave_settings_overrides.js

```diff
@@ -154,11 +154,18 @@ BravePatching.RegisterPolymerTemplateModifications({
     const appearanceBrowserEl = getMenuElement(templateContent, '/appearance')
     getStartedEl.insertAdjacentElement('afterend', appearanceBrowserEl)
     // Add Sync item
+    const isSyncEnabled = !loadTimeData.getBoolean('isSyncDisabled')
     const syncEl = createMenuElement(loadTimeData.getString('braveSync'), '/braveSync', 'brave_settings:sync')
```

Comment:
can we change it to 
``` patch
--- a/browser/resources/settings/brave_settings_overrides.js
+++ b/browser/resources/settings/brave_settings_overrides.js
@@ -153,12 +153,17 @@ BravePatching.RegisterPolymerTemplateModifications({
     // Move Appearance item
     const appearanceBrowserEl = getMenuElement(templateContent, '/appearance')
     getStartedEl.insertAdjacentElement('afterend', appearanceBrowserEl)
-    // Add Sync item
-    const syncEl = createMenuElement(loadTimeData.getString('braveSync'), '/braveSync', 'brave_settings:sync')
-    appearanceBrowserEl.insertAdjacentElement('afterend', syncEl)
     // Add Help Tips item
     const helpTipsEl = createMenuElement(loadTimeData.getString('braveHelpTips'), '/braveHelpTips', 'brave_settings:help')
-    syncEl.insertAdjacentElement('afterend', helpTipsEl)
+    // Add Sync item
+    const isSyncEnabled = !loadTimeData.getBoolean('isSyncDisabled')
+    if (isSyncEnabled) {
+      const syncEl = createMenuElement(loadTimeData.getString('braveSync'), '/braveSync', 'brave_settings:sync')
+      appearanceBrowserEl.insertAdjacentElement('afterend', syncEl)
+      syncEl.insertAdjacentElement('afterend', helpTipsEl)
+    } else {
+      appearanceBrowserEl.insertAdjacentElement('afterend', helpTipsEl)
+    }
     // Add Shields item
     const shieldsEl = createMenuElement(loadTimeData.getString('braveShieldsTitle'), '/shields',  'brave_settings:shields')
     helpTipsEl.insertAdjacentElement('afterend', shieldsEl)

```
? because we don't want to create `syncEl` when sync is disabled

Reply:
Updated. :)
I thought just creating an element doesn't affect much. Much better now!